docs/ref/models/relations.txt

```diff
@@ -135,12 +135,28 @@ Related objects reference
         :class:`~django.db.models.ForeignKey`\s where ``null=True`` and it also
         accepts the ``bulk`` keyword argument.
 
+    .. method:: set(obj1, [obj2, ...])
+
+    .. versionadded:: 1.8
+
+        Replace the set of related objects::
+
+            >>> new_list = [obj1, obj2, obj3]
+            >>> e.related_set.set(new_list)
```

Comment:
Ohh, probably to be consistent with .add(*object_list)? Still, it seems to me set(object_list) would be the most useful.